Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -470,28 +471,15 @@ class BrowserViewController: SensitiveViewController {
         }
     }
 
-    // Because crashedLastLaunch is sticky, it does not get reset, we need to remember its
-    // value so that we do not keep asking the user to restore their tabs.
-    var displayedRestoreTabsAlert = false
-
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
-        if !displayedRestoreTabsAlert && !cleanlyBackgrounded() && crashedLastLaunch() {
-            displayedRestoreTabsAlert = true
-            showRestoreTabsAlert()
-        } else {
-            tabManager.restoreTabs()
-        }
-
+        tabManager.restoreTabs()
```

Comment:
I think we still need to add the crash check back in (just not using sentry). Otherwise we can run into potential DoS attacks.